app/src/main/java/com/example/pygmyhippo/common/Entrant.java

```diff
@@ -31,7 +31,8 @@ public enum EntrantStatus {
         waitlisted("waitlisted"),
         cancelled("cancelled"),
         invited("invited"),
-        accepted("accepted");
+        accepted("accepted"),
+        rejected("rejected");
```

Comment:
same as james code?

Reply:
yeah, it event view was breaking since James had already added a few events with entrants as "rejected" in the database